docs/canvas/canvas-elements.asciidoc

```diff
@@ -0,0 +1,259 @@
+[role="xpack"]
+[[element-intro]]
+== Showcase your data with elements
+
+Canvas _elements_ are the building blocks of your workpad. With elements, you can combine images, text, and visualizations to tell a story about your data.
+
+When you add elements to your workpad, you can:
+
+* <<connect-element-data,Connect the elements to your live data>>
+
+* <<configure-display-options,Choose how you want the element to appear on your workpad>>
+
+* <<configure-auto-refresh-interval,Change the data refresh interval>>
+
+* <<organize-element,Move and arrange elements on your workpad>>
+
+* <<element-save,Save your elements>>
+
+[float]
+[[add-canvas-element]]
+=== Add elements to your workpad
+
+Choose the elements to display on your workpad, then familiarize yourself with the element using the preconfigured demo data.
+
+. Click *Add element*.
+
+. In the *Elements* window, select the element you want to use. 
++
+[role="screenshot"]
+image::images/canvas-element-select.gif[Canvas elements]
+
+. Play around with the default settings and see what the element can do. 
+
+TIP: Want to use a different element? You can delete the element by selecting it, clicking the *Element options* icon in the top right corner, then selecting *Delete*.
```

Comment:
Should we mention here that you can delete the element by hitting backspace (delete on Mac) as well?

Reply:
I don't want to confuse our users by telling them how to do something multiple ways. I chose this method because when they open the menu, they are exposed to `Cut`, `Copy`, `Paste`, and `Clone` as well.